server/src/test/java/org/opensearch/gateway/remote/RemoteClusterStateCleanupManagerTests.java

```diff
@@ -231,9 +288,26 @@ public void testDeleteClusterMetadata() throws IOException {
                     + ".dat"
             )
         );
+
+        verify(remoteRoutingTableService).deleteStaleIndexRoutingPaths(List.of(index3Metadata.getUploadedFilename()));
         Set<String> staleManifest = new HashSet<>();
         inactiveBlobs.forEach(blob -> staleManifest.add(new BlobPath().add(MANIFEST_PATH_TOKEN).buildAsString() + blob.name()));
         verify(container).deleteBlobsIgnoringIfNotExists(new ArrayList<>(staleManifest));
+
+        // Test case when remoteRoutingTableService is not enabled
+        container = mock(BlobContainer.class);
+        when(blobStore.blobContainer(any())).thenReturn(container);
+        doNothing().when(container).deleteBlobsIgnoringIfNotExists(any());
```

Comment:
Why do we need to mock these again?

Reply:
As we are verifying calls on container function `deleteBlobsIgnoringIfNotExists` function in UT, re-initialised container.

Reply:
You can reset mocks or verify it is called 2 times (or any number of times)